test/print_test_stats.py

```diff
@@ -125,6 +131,29 @@ def send_report(reports):
     )
     r.raise_for_status()
 
+def send_report_to_s3(reports):
+    sha1 = os.environ.get('CIRCLE_SHA1')
+    now = datetime.datetime.utcnow().isoformat()
+    key = f'test_time/{sha1}/{now}Z.json'  # Z meaning UTC
+    obj = boto3.resource('s3').Object('ossci-metrics', key)
```

Comment:
i might have less knowledge. but do we have credential to do so?

Reply:
I guess we'll find out once the CI tests finish for this PR :)

Reply:
CircleCI runners have an access key to some of S3 buckets in `fbossci` .

Reply:
@malfet the current commit is getting "Access Denied" errors when trying to upload; do I need to give additional permissions, since I created a new bucket just for this PR?